data/book_server.json

```diff
@@ -0,0 +1,2184 @@
+{
+    "source_analysis": {
+        "source": "Source code",
+        "target": [],
+        "appName": "java-example-app"
+    },
+    "source_analysis_on_bookserverapp": {
+        "source": "Source code",
+        "sources": ["java-ee", "springboot"],
+        "target": ["Linux", "Quarkus"],
+        "appName": "bookserver-app",
+        "file_name": "book-server",
+        "tags": ["EJB XML", "Java EE Batch", "EJB XML"],
+        "languages": ["Java"],
+        "issues": [
+            {
+                "name": "Add Maven profile to run the Quarkus native build",
+                "category": "mandatory",
+                "sources": ["java-ee"],
+                "targets": ["quarkus"],
+                "effort": 1,
+                "totalEffort": 1,
+                "incidents": 1,
+                "affectedFiles": 1,
```

Comment:
Files with incidents, line numbers, ideally also code snippet and at least begining of the incident message need to be verified too.  (then the single file might become too large).